taskcluster/focusios_taskgraph/__init__.py

```diff
@@ -19,7 +19,7 @@ def register(graph_config):
     Import all modules that are siblings of this one, triggering decorators in
     the process.
     """
-    _import_modules(["job"])
+    _import_modules(["job", "routes", "target_tasks"])
```

Comment:
This is now as in firefox-ios: https://github.com/mozilla-mobile/firefox-ios/blob/main/taskcluster/ffios_taskgraph/__init__.py